1주차/[Lv.3]_이중우선순위큐/주다현.js

```diff
@@ -0,0 +1,19 @@
+function solution(operations) {
+  let answer = [],
+    Q = [];
+
+  operations.forEach((v, i) => {
+    let [order, num] = v.split(' ');
```

Comment:
```suggestion
    const [order, num] = v.split(' ');
```
```javascript
// 1
const [order, num] = v.split(' ').map((v, i) => i ? Number(v): v);
// 2
const [order, num] = v.split(' ').map((v, i) => i ? +v : v);
// 3
const [order, num] = v.split(' ').map((v, i) => i ? parseInt(v) : v);
```
이런 방법도 있는 것 같아서 남겨봅니다!! ㅎㅎ 😁